Change so that all variables in initial algorithms are treated the same (#3562)
modelica/ModelicaSpecification: chapters/statements.tex

\item
  A continuous-time variable is initialized with the value of its \lstinline!start!-attribute.
\item

>   A discrete-time variable \lstinline!v! in a non-initial algorithm is initialized with \lstinline!pre(v)!.

Reviewer: Aren't we talking any algorithm when being part of the initialization problem?  To me, an _initial algorithm_ sounds lika an `initial algorithm`.  Hence:
```suggestion
  During transient analysis, a discrete-time variable \lstinline!v! in an algorithm is initialized with \lstinline!pre(v)!.
```

\item
  If at least one element of an array appears on the left hand side of the assignment operator, then the complete array is initialized in this algorithm section.
\item

>   In an initial algorithm, \cref{initialization-initial-equation-and-initial-algorithm}, any variable (including a parameter) is initialized with the value of its \lstinline!start!-attribute.

Reviewer: Similarly:
```suggestion
  Within the initialization problem treatment of an algorithm, \cref{initialization-initial-equation-and-initial-algorithm}, any variable (including a parameter) is initialized with the value of its \lstinline!start!-attribute.
```

Author: No, as indicated that change would be incorrect and break things.
